src/docs/user/references/SchedulerGraphQLAPI.adoc

```diff
@@ -0,0 +1,380 @@
+
+====  Scheduler GraphQL API portal
+
+ProActive Scheduler GraphQL API portal can be accessed in your environment using the following link:
```

Comment:
The Scheduler GraphQL API of ProActive can be ...
or 
The ProActive Scheduler GraphQL API can be ...

Reply:
I try to differenciate in this documentation the portal (which is like an IDE) to the rest api.